lib/puppet/provider/ec2_vpc/v2.rb

```diff
@@ -0,0 +1,94 @@
+require_relative '../../../puppet_x/puppetlabs/aws.rb'
+
+Puppet::Type.type(:ec2_vpc).provide(:v2, :parent => PuppetX::Puppetlabs::Aws) do
+  confine feature: :aws
+
+  mk_resource_methods
+  remove_method :tags=
+
+  def self.instances
+    regions.collect do |region|
+      response = ec2_client(region).describe_vpcs()
+      vpcs = []
+        response.data.vpcs.each do |vpc|
```

Comment:
Indendation is kind of weird here, I think this line is meant to be aligned with the line above, with the next line indented.